Formula/tin.rb

```diff
@@ -1,8 +1,8 @@
 class Tin < Formula
   desc "Threaded, NNTP-, and spool-based UseNet newsreader"
   homepage "http://www.tin.org"
-  url "https://www.nic.funet.fi/pub/unix/news/tin/v2.4/tin-2.4.5.tar.xz"
-  sha256 "1a14e6a49a3627230f792eb48936d39d0abd7b04d958083a6296bb4210c6c512"
+  url "https://www.nic.funet.fi/pub/unix/news/tin/v2.6/tin-2.6.0.tar.gz"
+  sha256 "efd97d9ed01797a2a2e475bcb7caac66eba2ad629f79384bfd26fb987740dabf"
```

Comment:
Should this be the more compressed `tar.xz` (1.5M) rather than `tar.gz` (3.2M)?

Reply:
make sense